src/ngraph_emitter.cc

```diff
@@ -1604,6 +1608,92 @@ void Emitter::CreateLayerOps() {
     }
     return conv;
   };
+
+  ngraph_op_funcs_["UpSampling"] =
+      [this](const NodePtr& node) -> NgraphNodePtr {
+
+    int scale = get_default(node, "scale", 1);
+    std::string sample_type =
+        get_default(node, "sample_type", std::string("nearest"));
+    std::string multi_input_mode =
+        get_default(node, "multi_input_mode", std::string("concat"));
+
+    NgraphNodePtr result{nullptr};
+    NgraphNodePtr input = op_map_.at(node->inputs_[0]);
+    if (sample_type == "nearest") {
+      auto upsample_nearest = [](const NgraphNodePtr& data,
+                                 const int scale) -> NgraphNodePtr {
```

Comment:
This is a pretty big lambda. Since you're not capturing any local state, does it make sense to pull it out into a helper function?